upgrade/upgrade-scripts/upgrade

```diff
@@ -99,25 +99,12 @@ function upgrade_in_place() {
 	"$IMAGE_PATH/upgrade-container" start "$CONTAINER" ||
 		die "failed to start '$CONTAINER'"
 
-	"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-		"$IMAGE_PATH/execute" -p "$(get_platform)" ||
-		die "'$IMAGE_PATH/execute' failed in '$CONTAINER'"
-
-	"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-		/bin/systemctl reload delphix-platform ||
-		die "'systemctl reload delphix-platform' failed in '$CONTAINER'"
-
-	"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-		/bin/systemctl restart delphix-platform ||
-		die "'systemctl restart delphix-platform' failed in '$CONTAINER'"
+	"$IMAGE_PATH/upgrade-container" upgrade "$CONTAINER" in-place ||
+		die "Failed to do in-place upgrade on '$CONTAINER'"
 
 	if [[ "$DLPX_UPGRADE_SKIP_VERIFY" != "true" ]]; then
-		"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-			"$IMAGE_PATH/verify-jar" "$@" ||
-			die "'$IMAGE_PATH/verify-jar' '$*' failed in '$CONTAINER'"
-		"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
-			"$IMAGE_PATH/verify-stack" "$@" ||
-			die "'$IMAGE_PATH/verify-stack' '$*' failed in '$CONTAINER'"
+		"$IMAGE_PATH/verify" -c "$CONTAINER" "$@" ||
```

Comment:
why use `verify` vs. `verify-jar` and `verify-stack`?

I ask, because I had originally thought that the `verify` script would eventually be removed, since the container creation code already exists here in the `upgrade` script.. and I was thinking the container creation logic would also get wrapped into the app-gate upgrade logic (and it'd use `verify-jar` and `verify-stack` directly).

I think I might have left a similar review on your app-gate change, so it might be good to discuss this at some point, so we're all on the same page.

Reply:
I initially thought of removing `verify` script as well since the verification logic is already defined in `upgrade` script, but I thought it would be important to maintain a way to run all verification logic (including container creation to cleanup) from a CLI since I believe there are other engineers who prefers doing that (although I'm not an avid user of this feature).

One benefit I noticed of having a separate `verify` script is that it simplifies client's code since it figures out which verification it needs to do, although this isn't a significant benefit now.

I'm fine with removing `verify` script

Reply:
Rather than having both an `upgrade` and `verify` script, how about we add a "dry-run" option to the existing `upgrade` script? I opened a PR to do that here: https://github.com/delphix/appliance-build/pull/243

My main concerns with the current `verify` script, is it has a lot of duplication between the `upgrade` script.. but more importantly, it can only be used to verify an in-place upgrade.. if we want a way to "verify" the upgrade, we should provide a way to do that for both "in-place" and "not-in-place" upgrades, which my proposed PR enables.

let me know what you think.

Reply:
Your PR looks great. I'll remove the `verify` script.